docs/src/plasma/qtplasmac.adoc

```diff
@@ -3800,38 +3756,27 @@ It will be referred to as "_xx" in this document, so "qtplasmac_xx.ts" in this d
 
 The default locale for QtPlasmaC is "_en" which means that any translation files created as "qtplasmac_en.*" will not be used for translations.
 
-If any of the required utilities (pyuic5, pylupdate5, linguist) are not installed then the user will need to install pyqt5-dev-tools:
+If any of the required utilities (pyuic5, pylupdate5, linguist) are not installed then the user will need to install pyqt5-dev-tools: +
+`sudo apt install pyqt5-dev-tools`
 
-----
-$ sudo apt install pyqt5-dev-tools
-----
+Change to the languages directory: +
+`cd ~/linuxcnc-dev/share/qtvcp/screens/qtplasmac/languages`
 
-Change to the languages directory:
-----
-$ cd ~/linuxcnc-dev/share/qtvcp/screens/qtplasmac/languages
-----
-
-If any text changes have been made to the GUI then run the following to update the GUI python file:
-----
-$ pyuic5 ../qtplasmac.ui > qtplasmac.py
-----
+If any text changes have been made to the GUI then run the following to update the GUI Python file: +
+`pyuic5 ../qtplasmac.ui > qtplasmac.py`
 
 The user can either create a new translation source file for a non-existing language translation
 or modify an existing translation source file due to changes being made to some text in a QtPlasmaC source file.
 If modifying an existing translation that has had no source file changes then this step is not required.
 
-Create or edit a .ts file:
-----
-$ langfile xx
-----
+Create or edit a .ts file: +
+`langfile xx`
 
 [NOTE]
 this command is a script which runs the following: $ pylupdate5 *.py ../*.py ../../../../../lib/python/qtvcp/lib/qtplasmac/*.py -ts qtplasmac_xx.ts
 
-The editing of the translation is done with the linguist application:
-----
-$ linguist
-----
+The editing of the translation is done with the linguist application: +
+`linguist`
```

Comment:
```suggestion
. Launch the Qt Linguist application
+
----
linguist
----
```

Reply:
I do not really like those blocks since everything in there is verbatim. You have no other style options like slanting arguments etc.